src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball() {
+    }
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public enum ColorList {

```

Comment:
enum name should represent what single element is
```suggestion
    public enum Color {

```